### PR TITLE
Multibrand GA Prod

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Updating UI content for multibrand GA prod.
- **Is this PR related to a Monolith release?** Yes. 2023.06.0

### Netlify links
- https://6480bc0c7c8fa5124da6c5aa--reverent-murdock-829d24.netlify.app/docs/guides/custom-url-domain/main/
- https://6480bc0c7c8fa5124da6c5aa--reverent-murdock-829d24.netlify.app/docs/guides/custom-widget/main/
- https://6480bc0c7c8fa5124da6c5aa--reverent-murdock-829d24.netlify.app/docs/guides/custom-error-pages/main/
- https://6480bc0c7c8fa5124da6c5aa--reverent-murdock-829d24.netlify.app/docs/guides/custom-email/main/

### Resolves:

* [OKTA-611526](https://oktainc.atlassian.net/browse/OKTA-611526)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/080d8df61b120d44d581d0f13c03846360f51861/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c41da7b7-cf40-4a6d-aaf3-85ce6c901079) | 66  | 96  | 93  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md](https://github.com/okta/okta-developer-docs/blob/080d8df61b120d44d581d0f13c03846360f51861/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-error-pages%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fe0e98a0-467b-4538-94bb-6d3848007690) | 53  | 85  | 66  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/080d8df61b120d44d581d0f13c03846360f51861/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/214fc969-c917-4b82-ba99-27c8ae0697c2) | 86  | 83  | 54  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md](https://github.com/okta/okta-developer-docs/blob/080d8df61b120d44d581d0f13c03846360f51861/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-widget%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2386e233-13a8-4349-8005-a9405d8f7344) | 61  | 76  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md](https://github.com/okta/okta-developer-docs/blob/080d8df61b120d44d581d0f13c03846360f51861/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-upgrade-sign-in-widget%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/154e8a9f-9f9d-47ee-b7a1-bb7440632630) | 31  | 89  | 82  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.1e4a0e81-1ea4-4efc-81f9-9ccf5c3164eb?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by arsalanbadar-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 1]
```diff
@@ -107,33 +105,25 @@ This method of configuring a custom domain is recommended because Okta manages y
...
 >  * If you have an Okta-managed certificate and you later get a CAA record, Okta can't renew your certificate. You must either add letsencrypt.org to the issuers list or remove the CAA record.
 
-1. In the Admin Console, go to **Customizations** > **Domain**. (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want.)
-2. In the **Custom URL Domain** box, click **Edit**. (If you enabled multibrand customization, on the **Domains** tab in the **Custom domain** section, click **Add domain**).
-3. Click **Get started**.
-4. On the **Add domain** page of the configuration wizard, in the **Certificate management** section, select **Okta-managed (faster and easier)**.
-
-### Add your subdomain information
-
-On the Add Domain page of the configuration wizard, enter your subdomain name, for example, `login.example.com`, and then click **Next**. Verifying domain ownership is the next step in the configuration wizard.
```
[the remark this answers is not in the document]
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:117]: I didn't remove it, I moved it to the section above :)

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 2]
```diff
@@ -75,9 +74,8 @@ Use these steps to add or edit a template in one of the Okta-supported languages
 
 > **Note:** To access email customization with a free developer edition of an Okta org, you need to contact [Okta support](https://support.okta.com/help).
 
-1. In the Admin Console, go to **Customizations** > **Branding**. (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want.)
-1. In the **Communications** section, in the **Emails** box, click **Edit**.
-1. In the **Email Templates** table, click a template name.
+1. In the Admin Console, go to **Customizations** > **Brands**, and then select the brand you want.
+1. In the **Emails** tab, click an email template name.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:78]: Acrolinx might want you to add a colon? If not, don't worry about this edit.

```suggestion
1. In the **Emails** tab, click an email template name:
```

**Review by vanngo-okta — APPROVED** [review]
LGTM 🚀

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 3]
```diff
@@ -75,9 +74,8 @@ Use these steps to add or edit a template in one of the Okta-supported languages
 
 > **Note:** To access email customization with a free developer edition of an Okta org, you need to contact [Okta support](https://support.okta.com/help).
 
-1. In the Admin Console, go to **Customizations** > **Branding**. (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want.)
-1. In the **Communications** section, in the **Emails** box, click **Edit**.
-1. In the **Email Templates** table, click a template name.
+1. In the Admin Console, go to **Customizations** > **Brands**, and then select the brand you want.
+1. In the **Emails** tab, click an email template name:
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:78]:
```suggestion
1. On the **Emails** tab, click an email template name:
```

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 4]
```diff
@@ -305,23 +295,29 @@ A custom email address allows you to present a branded experience to your end us
...
+### Use the Brands API
+
+The [Email Domains API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/EmailDomain/) is a feature that allows you to create, update, and delete email domains for your organization.
+
+### Configure a custom email address
+
+1. In the Admin Console, go to **Customizations** > **Brands**, and then select the brand you want.
 
-1. In the Admin Console, go to **Customizations** > **Emails**. (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want. In the **Domains** tab > **Email** section, click **Add domain**).
+2. In the **Domains** tab > **Emails** section, click **Add domain**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:306]:
```suggestion
2. On the **Domains** tab > **Emails** section, click **Add domain**.
```

File: packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md [hunk 5]
```diff
@@ -305,23 +295,29 @@ A custom email address allows you to present a branded experience to your end us
...
 
-2. In the **Email address** field, enter the email address that you want to send the system notification emails from. This email address appears in the emails sent to your users.
+3. In the **Email address** field, enter the email address that you want to send the system notification emails from. This email address appears in the emails sent to your users.
 
-3. In the **Name of email sender** field, enter the name of sender. This name appears as the sender in the emails sent to your users.
+4. In the **Name of email sender** field, enter the name of sender. This name appears as the sender in the emails sent to your users.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md:310]: trying to make this less "name of the sender" "email sender" "sender of the emails"
```suggestion
4. Enter the sender name in the **Name of email sender** field. This name appears as the sender of the emails sent to your users.
```

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 6]
```diff
@@ -54,9 +54,9 @@ The widget upgrade for a redirect sign-in flow depends on whether you configured
 
 To update the widget:
 
-1. In the Admin Console, go to **Customizations** > **Branding** then click **Edit** in the **Sign-in page** section. (If you enabled multibrand customization, go to **Customizations** > **Brands**, and then select the brand you want.)
+1. In the Admin Console, go to **Customizations** > **Brands**, and then select the brand you want.
 
-2. Go to the **Settings** tab. (If you enabled multibrand customization, in the **Pages** tab, click **Configure** for the page that has the version you want to upgrade.)
+2. In the **Pages** tab, click **Configure** in the **Sign-in page** section. Go to the **Settings** tab.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:59]:
```suggestion
2. On the **Pages** tab, click **Configure** in the **Sign-in page** section. Go to the **Settings** tab.
```

**Review by susanharper-okta — APPROVED** [review]
couple baby nits - approving to not hold you up.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/e5bb7157b268e1d427fac20c30409af97c7fe574/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/df16a581-c7b5-4e9c-98d4-4f3798fe57d2) | 66  | 96  | 93  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md](https://github.com/okta/okta-developer-docs/blob/e5bb7157b268e1d427fac20c30409af97c7fe574/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-error-pages%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bcf9a727-7662-4d58-b449-3fb81d14d486) | 53  | 85  | 66  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/e5bb7157b268e1d427fac20c30409af97c7fe574/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/254ce1b1-d843-4fc8-8eb8-37549d14fe54) | 86  | 83  | 54  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md](https://github.com/okta/okta-developer-docs/blob/e5bb7157b268e1d427fac20c30409af97c7fe574/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-widget%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/97da6c86-a09f-4acc-836c-a967fa0a50ab) | 61  | 76  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md](https://github.com/okta/okta-developer-docs/blob/e5bb7157b268e1d427fac20c30409af97c7fe574/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-upgrade-sign-in-widget%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b3366389-6143-4905-91b4-dcebcd22bf88) | 31  | 89  | 82  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.9f537970-7695-487f-bb89-a0079b0a7752?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md](https://github.com/okta/okta-developer-docs/blob/92d62b392d7f09532c2d88f862fc316a2ad33174/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-email%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/e1a9b345-2714-43f7-bf08-51faf7454adf) | 66  | 96  | 93  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md](https://github.com/okta/okta-developer-docs/blob/92d62b392d7f09532c2d88f862fc316a2ad33174/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-error-pages%2Fmain%2Findex.md) | [78](https://okta.acrolinx.cloud/api/v1/checking/scorecards/5c0fb9e9-0489-4ec4-b0e1-51a3fb1b8ce9) | 53  | 85  | 66  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md](https://github.com/okta/okta-developer-docs/blob/92d62b392d7f09532c2d88f862fc316a2ad33174/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-url-domain%2Fmain%2Findex.md) | [81](https://okta.acrolinx.cloud/api/v1/checking/scorecards/32e2f7c6-d5e4-4dc7-ae2f-06a6453f1d48) | 86  | 84  | 54  | 99  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md](https://github.com/okta/okta-developer-docs/blob/92d62b392d7f09532c2d88f862fc316a2ad33174/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fcustom-widget%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/15a7ba09-2e9c-4ce5-880c-737445f732d6) | 61  | 76  | 61  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md](https://github.com/okta/okta-developer-docs/blob/92d62b392d7f09532c2d88f862fc316a2ad33174/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-upgrade-sign-in-widget%2Fmain%2Findex.md) | [79](https://okta.acrolinx.cloud/api/v1/checking/scorecards/736b25e0-eb93-41d7-b060-4f3ef74f8b46) | 31  | 89  | 82  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.6d92aaf8-1e34-4c72-bd0b-3271964fa832?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**